src/components/DropdownList.tsx

```diff
@@ -0,0 +1,45 @@
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  onHover: (val: boolean) => void;
+  onSelect?: (person: Person) => void;
+  onPersonSelect?: (name: string) => void;
+};
+
+export const DropdownList: React.FC<Props> = ({
+  people,
+  onHover = () => {},
```

Comment:
```suggestion
  onHover,
```
It is required property according to Props type